blog/_posts/2021-10-19-scala-3.1.0-released.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.0 released!
+---
+Hello from the Scala 3 team! It has already been six weeks since we have announced the release candidate for the first minor version after the initial release of Scala 3. Now, after two more RCs, we can confidently promote Scala 3.1.0-RC3 to a new stable release of the language. In other words, Scala 3.1.0 is officially out!
+
+You don't need to worry about having to migrate your projects to the new version. Scala 3 has strong compatibility guarantees, and all code working in 3.0.2 will also work in 3.1. On the other hand, switching to Scala 3.1 allows you to use many improvements and a handful of newly stabilized APIs.
```

Comment:
I think "worry when upgrading your projects" would be clearer than "worry about having to migrate your projects"

Reply:
I wouldn't use worry at all and maintain a more cold tone.

> Thanks to Scala 3's strong compatibility guarantees, all code 3.0.2 will work in 3.1 However ....